### PR TITLE
feat(cli): add `--version` and `--message` flags to `app deploy`

### PR DESCRIPTION
**Pull request by jfeodor**
Allow marking deployed versions with a version number and a message.

### CODE REVIEW

File: cli/cmd/app/deploy/deploy_test.go [hunk 1]
```diff
@@ -32,32 +29,36 @@ func TestDeploy(t *testing.T) {
 		apps.On("DeployApp", mock.Anything, mock.Anything).Return(app.DeployAppOutput{DeploymentVersionID: deployVersionID}, nil)
 		apps.On("DeployEvents", mock.Anything, mock.Anything).Return(nil)
 
-		err := Deploy(context.TODO(), apps, appDir, "", slug, appName, false)
+		return apps
+	}
+
+	t.Run("give no existing app then happy path can run", func(t *testing.T) {
```
Comment (moroderNumerous) [inline cli/cmd/app/deploy/deploy_test.go:35]: Seems like this and the next test are identical tests with different names

**Review by moroderNumerous — COMMENTED** [review]
looks good, theres just this one thing with tests, other than that its 🥇

File: cli/cmd/app/deploy/deploy_test.go (hunk 1 above)
Comment (jfeodor) [reply cli/cmd/app/deploy/deploy_test.go:35]: Nice catch! It was a mistake in the refactoring of the setup.

Comment (jfeodor) [reply cli/cmd/app/deploy/deploy_test.go]: 96d59abc065010c4506235feece548a246ed869e

**Review by moroderNumerous — APPROVED** [review]

**Pull request merged**